modules/actor/sheet/actor-sheet.js

```diff
@@ -1130,14 +1130,14 @@ export default class ActorSheetWfrp4e extends ActorSheet {
     }
     renderTemplate('systems/wfrp4e/templates/dialog/delete-item-dialog.html').then(html => {
       new Dialog({
-        title: "Delete Confirmation", content: html, buttons: {
+        title: game.i18n.localize("Delete Confirmation"), content: html, buttons: {
```

Comment:
This key isn't in `en.json` and I doubt it's provided by Foundry, if it is though this can just be resolved.